GliaWidgetsTests/ViewModel/Chat/ChatItemTests.swift

```diff
@@ -0,0 +1,60 @@
+@testable import GliaWidgets
+import XCTest
+
+final class ChatItemTests: XCTestCase {
+    func testIsOperatorReturnsTrueForExpectedKindCases() {
+        let callUpgrade = ChatItem.Kind.callUpgrade(.init(with: .audio), duration: .init(with: .zero))
+        let choiceCard = ChatItem.Kind.choiceCard(.mock(), showsImage: false, imageUrl: "", isActive: false)
+        let customCard = ChatItem.Kind.customCard(.mock(), showsImage: false, imageUrl: nil, isActive: false)
+        let operatorConnected = ChatItem.Kind.operatorConnected(name: nil, imageUrl: nil)
+        let queueOperator = ChatItem.Kind.queueOperator
+        let outgoingMessage = ChatItem.Kind.outgoingMessage(.mock())
+        let operatorMessage = ChatItem.Kind.operatorMessage(.mock(), showsImage: false, imageUrl: nil)
+        let systemMessage = ChatItem.Kind.systemMessage(.mock())
+        let unreadMessageDivider = ChatItem.Kind.unreadMessageDivider
+        let visitorMessage = ChatItem.Kind.visitorMessage(.mock(), status: nil)
+        let transferring = ChatItem.Kind.transferring
+        let kinds: [ChatItem.Kind] = [
+            callUpgrade,
+            choiceCard,
+            customCard,
+            operatorConnected,
+            queueOperator,
+            outgoingMessage,
+            operatorMessage,
+            systemMessage,
+            unreadMessageDivider,
+            visitorMessage,
+            transferring
+        ]
+        for kind in kinds {
+            let chatItem = ChatItem(kind: kind)
+            switch kind {
+            case .callUpgrade:
```

Comment:
I think the ones that need to be true can be grouped into one case, no? and then the same for the false ones

Reply:
Indeed a84c9d6236615aba80dddccce0a80b7dd616247f.